india_stocks_api/brokers/angel.py

```
@@ -201,92 +442,516 @@ def get_quote(self, instrument: Equity | Future | Option | Index) -> dict:
+            )
+
+    def cancel_order(self, order_id: str) -> OrderExecutionResponse:
         jwt_token = context.get_auth_token()
+        if not jwt_token:
+            raise RuntimeError("Auth failed")
         
-        # 1. Find Order to get Symbol/Exchange
-        all_orders = self.get_orders()
-        target_order = next((o for o in all_orders if o.get('orderid') == order_id), None)
+        try:
+            resp, status = cancel_order_api(order_id, jwt_token)
```

Ruff (0.15.1)

[warning] Unpacked variable `status` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)